tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>I don't actually think you need this section now, at this point, you've reached to goal of the tutorial. Maybe you could <em>show</em> the different documents you've now indexed which you have in the document store, and indicate people to the RAG tutorial as a next step (which you'ce done at the bottom).  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I kinda like keeping the section below. that provides a whole end to end "solution" and proves that the documents were properly indexed. What about a compromise - something like:</p><p> </p><p><code>If you only wanted to learn how to preprocess documents, you can stop here! If you want to see an example of using those documents in a RAG pipeline, read on.  </code> </p>

Reply:
Yeee let's do that.. one suggestion then, but I'll leave it to you: I noticed people appreciate open source model examples, and I think they'd work well here. Maybe try using HuggingFaceTGIGenerator?

Reply:
<p>actually, none of the open source models did a good job at this. </p><p> </p><p>{'llm': {'replies': ['\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n'],</p><p>  'meta': [{'model': 'mistralai/Mistral-7B-v0.1',</p><p>    'index': 0,</p><p>    'finish_reason': 'length',</p><p>    'usage': {'completion_tokens': 20,</p><p>     'prompt_tokens': 3452,</p><p>     'total_tokens': 3472}}]}}</p><p> </p><p> </p><p>{'llm': {'replies': [' To make vegan keto eggplant lasagna, you would need 2 large eggplants'],</p><p>  'meta': [{'model': 'mistralai/Mixtral-8x7B-Instruct-v0.1',</p><p>    'index': 0,</p><p>    'finish_reason': 'length',</p><p>    'usage': {'completion_tokens': 20,</p><p>     'prompt_tokens': 3452,</p><p>     'total_tokens': 3472}}]}}</p><p> </p><p>{'llm': {'replies': [' For vegan keto eggplant lasagna, you would need 2 large eggplants,'],</p><p>  'meta': [{'model': 'mistralai/Mistral-7B-Instruct-v0.2',</p><p>    'index': 0,</p><p>    'finish_reason': 'length',</p><p>    'usage': {'completion_tokens': 20,</p><p>     'prompt_tokens': 3452,</p><p>     'total_tokens': 3472}}]}}</p><p> </p><p>{'llm': {'replies': [' You would need sunflower seeds, hemp hearts, miso paste, nutritional'],</p><p>  'meta': [{'model': 'codellama/CodeLlama-34b-Instruct-hf',</p><p>    'index': 0,</p><p>    'finish_reason': 'length',</p><p>    'usage': {'completion_tokens': 20,</p><p>     'prompt_tokens': 3535,</p><p>     'total_tokens': 3555}}]}}</p>

Reply:
resolved this by increasing the generation kwargs 👍